config/300-apiserversource.yaml

```diff
@@ -0,0 +1,166 @@
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          properties:
+            serviceAccountName:
+              type: string
+              description: "name of the ServiceAccount to use to run the receive adapter. More info: https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/."
+            sink:
+              anyOf:
+              - type: object
+                description: "the destination that should receive events."
+                properties:
+                  ref:
+                    type: object
+                    description: "a reference to a Kubernetes object from which to retrieve the target URI."
+                    required:
+                    - apiVersion
+                    - kind
+                    - name
+                    properties:
+                      apiVersion:
+                        type: string
+                        minLength: 1
+                      kind:
+                        type: string
+                        minLength: 1
+                      name:
+                        type: string
+                        minLength: 1
+                  uri:
+                    type: string
+                    description: "the target URI. If ref is provided, this must be relative URI reference."
+              - type: object
+                description: "DEPRECATED: a reference to a Kubernetes object from which to retrieve the target URI."
+                required:
+                - apiVersion
+                - kind
+                - name
+                properties:
+                  apiVersion:
+                    type: string
+                    minLength: 1
+                  kind:
+                    type: string
+                    minLength: 1
+                  name:
+                    type: string
+                    minLength: 1
+            mode:
+              type: string
+              description: "Mode controls the content of the event payload. One of: 'Ref' (only references of resources), 'Resource' (full resource)."
+            resources:
+              items:
+                properties:
+                  apiVersion:
+                    type: string
+                    description: "API version of the objects to watch."
+                  kind:
+                    type: string
+                    description: "Kind of the objects to watch."
+                  controller:
+                    type: boolean
+                    description: "If true, emits the managing controller ref. Only supported for mode=Ref. More info: https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/"
+                  labelSelector:
+                    type: object
+                    description: "Label selector restricting this  list of objects to watch by their labels. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/"
```

Comment:
nit, extra spaces between 'this' and 'list'

Reply:
done.